plugin/src/main/java/org/opensearch/ml/task/MLTaskDispatcher.java

```diff
@@ -74,7 +74,7 @@ public void dispatch(ActionListener<DiscoveryNode> actionListener) {
 
     public void dispatchPredictTask(String[] nodeIds, ActionListener<DiscoveryNode> actionListener) {
         if (nodeIds == null || nodeIds.length == 0) {
-            throw new IllegalArgumentException("Model not loaded yet");
+            throw new IllegalArgumentException("no eligible node to run predict request");
```

Comment:
Is it also possible to enter this branch if model not loaded?

Reply:
No, it will fail earlier.